.github/workflows/ci.yml

```diff
@@ -53,10 +53,6 @@ jobs:
         working-directory: zenoh-jni
         run: cargo clippy --all-targets --all-features -- -D warnings
 
-      - name: Check for feature leaks
```

Comment:
There are no native tests. Testing is done through the Kotlin layer that invokes the library. Therefore this step only causes the CI to take more time, because it builds the code again... Maybe this deserves a PR on its own.

Reply:
That test is in place to ensure features between `zenoh` and `zenoh-jni` don't leak.
There is one [test](https://github.com/eclipse-zenoh/zenoh-kotlin/blob/3e88902415e36b885fabb809aecb3737e595885e/zenoh-jni/src/lib.rs#L30) that checks that in the same is checked in all other API bindings.

Reply:
Ooh I see that's something you just added today! My bad, I'll revert this, it's just I was unaware of your PR and I thought it was a mistake on my part I had committed in the past when writing the CI yml file.

Reply:
No problem, misunderstanding clarified!